common/geometry/translate_types_test.cc

```diff
@@ -0,0 +1,37 @@
+#include "common/geometry/translate_types.hh"
+
+#include "gtest/gtest.h"
+
+namespace robot::geometry {
+TEST(TranslateTypesTest, eigen_mat_and_cv) {
+    int rows = 5, cols = 5;
+    Eigen::MatrixXd mat_eig = Eigen::MatrixXd::Random(rows, cols);
```

Comment:
It doesn't really matter in this case, but in general, it's best practice to avoid generating random numbers in a unit test as it can make the tests flaky. Consider hard coding a test input here and in the test below.